src/components/tabs/index.d.ts

```diff
@@ -37,4 +39,5 @@ declare module '@elastic/eui' {
   export const EuiTabbedContent: FunctionComponent<
     EuiTabbedContentProps & CommonProps & HTMLAttributes<HTMLDivElement>
   >;
+  export const EuiTab: FunctionComponent<EuiTabProps>;
```

Comment:
Because the `tabs/` directory isn't entirely TypeScript yet, we still need to export all the components from here so they can be picked up properly when imported.

Once all of `tabs/` has been converted, this file will be removed.